tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/util/TlaExUtil.scala

```diff
@@ -1,13 +1,18 @@
 package at.forsyte.apalache.tla.bmcmt.util
 
 import at.forsyte.apalache.tla.bmcmt.InvalidTlaExException
-import at.forsyte.apalache.tla.lir.oper.{ApalacheOper, TlaActionOper}
+import at.forsyte.apalache.tla.lir.oper.{ApalacheOper, TlaActionOper, TlaOper}
 import at.forsyte.apalache.tla.lir.{LetInEx, NameEx, OperEx, TlaEx, TlaOperDecl}
 
 object TlaExUtil {
+  // Several Apalache operators accept definitions as arguments. Their variables are not considered.
```

Comment:
I don't get this comment. It's true, but it feels like a non-sequitur (because the method is not `acceptsDefinitions` but `isFold`)

Reply:
Right. I have changed the name later